### PR TITLE
chore: upgrade to latest of all dependencies

### PR DESCRIPTION
**Pull request by JustinBeckwith**

### CODE REVIEW

File: test/util.ts [hunk 1]
```diff
@@ -1761,14 +1755,15 @@ describe('common/util', () => {
     // in node v4 which is why the eval call is wrapped in a try catch block.
     try {
       eval(`
+        const assert2 = require('assert');
```
Comment (ofrobots) [inline test/util.ts]: Why require assert again here?

File: src/service.ts [hunk 2]
```diff
@@ -18,13 +18,13 @@
...
-import * as arrify from 'arrify';
+import arrify from 'arrify';
 import {Duplexify} from 'duplexify';
-import * as extend from 'extend';
+import extend from 'extend';
 import {GoogleAuth} from 'google-auth-library';
-import * as is from 'is';
-import * as pify from 'pify';
-import * as r from 'request';
+import is from 'is';
```
Comment (ofrobots) [inline src/service.ts]: not a comment for this PR, but if possible, we should switch to `@sindresorhus/is`. I find that `is` has non-intuitive behavior, e.g. `is.object(new Error())` returns false.

Comment (JustinBeckwith) [reply src/service.ts]: :nods: I opened #163 to track

File: test/util.ts (hunk 1 above)
Comment (JustinBeckwith) [reply test/util.ts]: There's some weirdness with the combination of synthetic imports and the eval.  I get a `ReferenceError: assert is not defined` if I don't re-import it using `require`.  Not entirely sure what's happening.

**Comment by JustinBeckwith** [comment]
*gentle ping* @alexander-fenster @stephenplusplus @ofrobots

**Review by alexander-fenster — APPROVED** [review]

**Pull request merged**